spec/truffle.mspec

```diff
@@ -79,16 +79,19 @@ class MSpecScript
     # Hangs in CI
     "^spec/ruby/library/net",
 
-    # openssl is tested separately as it needs Sulong
+    # openssl and zlib are tested separately as they need Sulong
     "^spec/ruby/library/openssl",
+    "^spec/ruby/library/zlib"
   ]
 
   set :capi, [
     "spec/ruby/optional/capi"
   ]
 
+  # TODO (pitr-ch 11-Jan-2018): rename to library_cext
```

Comment:
Is there any reason not to rename it yet and have a misleading name?
It's just changing one line in `mx.truffleruby/mx_truffleruby.py`, isn't it?

Reply:
Calling it `:openssl` when it does more than OpenSSL is the part he wants to fix.

Reply:
Yeah, so why not do it right away so there is no confusing name in `master`?

Reply:
Because it can break CIs and I wanted to do it in another PR to separate the change. It will be in a follow up PR.